testing/camunda-process-test-java/src/main/java/io/camunda/process/test/api/CamundaProcessTestExtension.java

```diff
@@ -337,6 +351,10 @@ public void afterAll(final ExtensionContext context) throws Exception {
       // Skip if the runtime is not created.
       return;
     }
+    if (!context.getUniqueId().equals(initializingExtensionContextId)) {
```

Comment:
🔧 We can replace the context ID with an annotation check.

```suggestion
    if (!AnnotationUtils.isAnnotated(context.getTestClass(), CamundaProcessTest.class)) {
```

We could use the same check for the coverage test name.